src/rules/enforce-id-capitalization.ts

```
@@ -0,0 +1,84 @@
+
+/**
+ * This rule ensures consistency in user-facing text by enforcing the use of "ID"
+ * instead of "id" when referring to identifiers in UI labels, instructions,
+ * error messages, and other visible strings.
+ */
+export const enforceIdCapitalization = createRule<Options, MessageIds>({
+  name: 'enforce-id-capitalization',
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Enforce the use of "ID" instead of "id" in user-facing text',
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)